utils/checks.stanza

```diff
@@ -54,14 +57,15 @@ public defn erc-check-design (module:JITXObject):
   inside pcb-module:
     for i in populated-components(module) do :
       check rated-temperature(i)
-      check capacitor-component(i) when has-property?(i.capacitor)
-      check resistor-component(i)  when has-property?(i.resistor)
-      check inductor-component(i)  when has-property?(i.inductor)
-      check diode-component(i)     when has-property?(i.diode)
```

Comment:
This is not an accident: the diode component checks are redundant (only pin connectivity is checked, which is handled elsewhere)

Reply:
We should add 1-2 more diode checks (like reverse breakdown voltage, max forward current) but your simplification is ok. Let's plan to add diodes to the passive-checks.